source/adminguide/accounts.rst

```diff
@@ -807,4 +807,30 @@ The admin can also disable 2FA for a User using the action button as shown below
           If the admin themself loses the authenticator application or forgets the static PIN, then the admin
           will have to either use apikey to disable 2FA using the API setupUserTwoFactorAuthentication with
           enable flag to false or to do the database changes in 'user' table by clearing the columns
-          'is_user_2fa_enabled', 'key_for_2fa', 'user_2fa_provider' for the specific entry.
\ No newline at end of file
+          'is_user_2fa_enabled', 'key_for_2fa', 'user_2fa_provider' for the specific entry.
+
+Setting up SMTP for Password Recovery
+--------------------------------------
+
+CloudStack supports password recovery using email. To enable this feature,
+set global setting `user.password.reset.enabled` to `true`. The following
+global settings are available to configure SMTP for password recovery.
+
+.. cssclass:: table-striped table-bordered table-hover
+
+================================================   ================   ===================================================================
+Global setting                                     Default values     Description
+================================================   ================   ===================================================================
+user.password.reset.enabled                        false              Determines whether password recovery via email is enabled or not
```

Comment:
"user.password.reset.mail.template" setting is missing here, please check if anything else needs to be added.